views/ourteam.hbs

```diff
@@ -0,0 +1,48 @@
+  <div id="the-team" class="bg-white">
+        <div class="our-team-contain">
+          <div class="team-block">
+            <div class="team-left">
+              <div class="team-image"><img src="/images/ourteam/honree.png" alt="honree"></div>
+            </div><!-- end .team-left -->
+            <div class="team-right">
+              <div class="team-text">
+                <h2>Honree <span>Creative Director</span></h2>
+                <div class="line-sep"></div>
+                <p>Like a baby bird taking flight Ben has grown Elegant Seagulls from a one-man operation to one of the largest agencies in MI. As creative director he oversees all things design. When not cracking hilarious jokes Ben enjoys surfing, snowboarding, CrossFit, and spending time with his family and two dogs.</p>
+               
+              </div><!-- end .team-text -->
+            </div><!-- end .team-right -->
+            <div class="clear"></div>
+          </div><!-- end .team-block -->
+
+          <div class="team-block">
+            <div class="team-right">
+              <div class="team-image"><img src="/images/ourteam/bloke2.png" alt="bloke"></div>
```

Comment:
this WILL not prevent my flawless commit. please merge.

On Mon, Mar 23, 2015 at 11:20 PM, Blake Parkinson <notifications@github.com>
wrote:

> In views/ourteam.hbs
> <https://github.com/blakeparkinson/sports2/pull/191#discussion_r27005206>:
>
> > +              <div class="team-image"><img src="/images/ourteam/honree.png" alt="honree"></div>
> > +            </div><!-- end .team-left -->
> > +            <div class="team-right">
> > +              <div class="team-text">
> > +                <h2>Honree <span>Creative Director</span></h2>
> > +                <div class="line-sep"></div>
> > +                <p>Like a baby bird taking flight Ben has grown Elegant Seagulls from a one-man operation to one of the largest agencies in MI. As creative director he oversees all things design. When not cracking hilarious jokes Ben enjoys surfing, snowboarding, CrossFit, and spending time with his family and two dogs.</p>
> > +
> > +              </div><!-- end .team-text -->
> > +            </div><!-- end .team-right -->
> > +            <div class="clear"></div>
> > +          </div><!-- end .team-block -->
> > +
> > +          <div class="team-block">
> > +            <div class="team-right">
> > +              <div class="team-image"><img src="/images/ourteam/bloke2.png" alt="bloke"></div>
>
> Can i get you a better shirtless pic? This one is a bad angle and i look
> flabby
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/191/files#r27005206>.
>



-- 
*Henry Lyford*
Manager, Partnerships & Business Development
Edmodo.com/henry